Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -87,7 +88,8 @@ class TabScrollingController: NSObject {
     private var scrollDirection: ScrollDirection = .Down
     private var toolbarState: ToolbarState = .Visible
 
-    override init() {
+    init(profile: Profile) {
```

Comment:
Very strange, I'm 100% certain I had done the work to prevent hiding the toolbars!
I must have missed something when transferring my WIP into my newly-created repo. 😢